blueman/gui/Notification.py

```diff
@@ -167,10 +167,18 @@ class Notification(object):
     def actions_supported():
         return "actions" in Notify.get_server_caps()
 
+    @staticmethod
+    def daemon_available():
+        return Notify.get_server_caps() != []
```

Comment:
Maybe we should check `'body' in Notify.get_server_caps()`. That's the minimal capability we need for useful notifications and I'm not sure if it would always return `[]` if the daemon is not available (could possibly `None` or something...).